tiltak/src/main/kotlin/no/nav/amt/tiltak/deltaker/service/DeltakerServiceImpl.kt

```diff
@@ -433,6 +433,23 @@ open class DeltakerServiceImpl(
 		log.info("Ferdig med republisering av deltakere på kafka")
 	}
 
+	override fun republiserDeltakerePaDeltakerV2(tiltakstype: String) {
+		if (unleashService.erKometMasterForTiltakstype(tiltakstype)) {
+			log.info("Republiserer ikke deltakere for tiltakstype $tiltakstype siden komet er master for tiltakstypen")
```

Comment:
Vet ikke helt om denne guarden er nødvendig, kunne vært fint at den som kaller bestemmer utifra behov, uavhengig av om vi er master eller ikke

Reply:
Hvis vi ikke sjekker det her så blir det uansett sjekket av publisheren pr deltaker lenger ned. Jeg mener fortsatt at det er feil at amt-tiltak skal publisere deltakere den ikke er master for - da vil den f.eks. overskrive felter den ikke kjenner til med null, f.eks. historikk.